templates/web/base/admin/problem_row.html

```diff
@@ -29,13 +29,15 @@
                 [%- PROCESS value_or_nbsp value=problem.bodies_str -%]
             [%- END -%]
             <br>[% problem.cobrand %]<br>[% problem.cobrand_data | html %]
-            <br>
-            [% IF c.user.has_body_permission_to('assign_report_to_user') %] [%# see report's assigned user %]
-                <span>[% loc('Assigned to') %]: </span>
-                <span>
-                    [%- problem.shortlisted_user.name OR problem.shortlisted_user.username OR loc('unassigned') -%]
-                </span>
-            [%- END -%]
+            [% IF !c.cobrand.moniker == 'cheshireeast' %]
```

Comment:
Similar to the Perl module, is there a way we can remove references to an individual cobrand here (and in the other template)? They're so enormous that you couldn't just override the entire template.